### ISSUE TITLE
How to add size,site to webhose search

### ISSUE THREAD

**Issue by gopighub** [Buzzilla/webhose-python#5]

**Comment by rangeva** [comment]
Hi,
The Query object has a site parameter that can either be a list (for multiple sites) or a string (for a single site). Here is the piece of code that is responsible to build the query out of this parameter:

`if self.site:
            if type(self.site) is list:
                qs.append(" OR ".join("site:%s" % site for site in self.site))
            else:
                qs.append("site:%s" % self.site)`

**Comment by gopighub** [comment]
Thanks for your reply.
It is possible to use 'size' parameter in Query object?

**Comment by rangeva** [comment]
No. The size parameter isn't supported by the current version, but you are more than welcome to update it and send a push request :)
